enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/scenarios/ClusterOverviewIT.java

```diff
@@ -266,6 +269,21 @@ public void shouldDiscoverRemovalOfCoreMembers() throws Exception
         }
     }
 
+    @Test
+    public void shouldHaveNoLeaderIfMajorityCoreMembersDead() throws Exception
+    {
+        clusterRule.withNumberOfCoreMembers( 3 );
+        clusterRule.withNumberOfCoreMembers( 2 );
+
+        Cluster cluster = clusterRule.startCluster();
+        //TODO: address proliferation of Role enums
```

Comment:
Maybe we should have a Trello lane with refactoring tasks

Reply:
Done